glean-core/src/metrics/mod.rs

```diff
@@ -173,6 +180,9 @@ impl Metric {
             Metric::Datetime(d, time_unit) => json!(get_iso_time_string(*d, *time_unit)),
             Metric::Experiment(e) => e.as_json(),
             Metric::Quantity(q) => json!(q),
+            Metric::Rate(num, den) => {
+                json!({"numerator": num, "denominator": den})
```

Comment:
Didn't we want to explore if we can skip this metric if denominator is 0, because that's not particular helpful?

Reply:
If the numerator is 0, I think you mean. And yeah, we talked about it, but I thought we'd settled on "keep it simple for now, and optimize it later as needed". If we get 2093 use counters with 0 for their numerators and it's blowing up our user disk/user bandwidth/moz bandwidth/moz disk budget, then we can adjust then.

Reply:
No, I meant the denominator.
If that's never increased we end up with a rate such as 17/0. Is that useful? (I would need to re-read the proposal, but we discussed it there)

Reply:
Then the next ping has 0/17. We need 17/0 from the first ping to aggregate them together.

Reply:
Oh.